baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/ProxyNonConstantType.java

```diff
@@ -38,7 +38,7 @@
         name = "ProxyNonConstantType",
         link = "https://github.com/palantir/gradle-baseline#baseline-error-prone-checks",
         linkType = LinkType.CUSTOM,
-        severity = SeverityLevel.SUGGESTION,
+        severity = SeverityLevel.WARNING,
```

Comment:
Do you know how many repos have hit this?

Reply:
A few will hit it hard, most don't hit it at all. I suppressed the warning in WC where we hit it on an old retrofit codepath.